.changeset/cold-zoos-wash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cfworker/cosmos': patch
```

Comment:
@ItalyPaleAle thoughts? Initially was 1.12.2, the masterKey->accountKey was 1.13.0, this would make 1.13.1.

Reply:
As long as it's compatible with previous 1.x releases, this is fine.

Reply:
Ok I'm going to rename this back to masterKey and unpublish 1.13.0 as you've recommended. Sorry for the kerfuffle.